Packages/io.chainsafe.web3-unity.marketplace/package.json

```diff
@@ -23,5 +23,11 @@
         "email": "bd@chainsafe.io",
         "url": "https://gaming.chainsafe.io/"
     },
-    "samples": []
+    "samples": [
```

Comment:
When adding samples you need to also update the CI duplicate text file with new directories